scripts/testnet/setup_stride_state.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+set -eu
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+NUM_NODES="$1"    
+
+NETWORK_NAME=stride
+CHAIN_NAME=STRIDE
+VAL_PREFIX=val
+VAL_TOKENS=500000000ustrd
+STAKE_TOKENS=300000000ustrd
+
+PEER_NODE_IDS=""
+MAIN_ID=1 # Node responsible for genesis and persistent_peers
+MAIN_NODE_NAME=""
+MAIN_NODE_CMD=""
+echo 'Initializing stride...'
+for (( i=1; i <= $NUM_NODES; i++ )); do
+    # Node names will be of the form: "stride-node1"
+    node_name="${NETWORK_NAME}-node${i}"
+    # Moniker is of the form: STRIDE_1
+    moniker=$(printf "${NETWORK_NAME}_${i}" | awk '{ print toupper($0) }')
+
+    # Create a state directory for the current node and initialize the chain
+    mkdir -p $STATE/$node_name
+    st_cmd="$STRIDE_CMD --home ${STATE}/$node_name"
+    $st_cmd init $moniker --chain-id $CHAIN_NAME --overwrite 2> /dev/null
+
+    # Update node networking configuration 
+    sed -i -E "s|cors_allowed_origins = \[\]|cors_allowed_origins = [\"\*\"]|g" "${STATE}/${node_name}/config/config.toml"
```

Comment:
good flag on this yesterday, will see if we can restrict this to only stridenet.co